build.md

```diff
@@ -155,6 +155,39 @@ args:
   - GIT_COMMIT
 ```
 
+### ssh
+
+`ssh` defines an SSH authentication that the image builder SHOULD use during image build (e.g., cloning private repository)
+
+`ssh` property syntax can be either:
+* `default` - let the builder connect to the ssh-agent.
+* `ID=path` - a key/value definition of an ID and the associated path. Can be either a [PEM](https://en.wikipedia.org/wiki/Privacy-Enhanced_Mail) file, or path to ssh-agent socket
+* `{}` (no value) which is a shortcut of the `default` configuration
```

Comment:
sounds a bit weird to me setting `ssh: {}` actually set a value, while - from a yaml point of view - we try to force `null` value, which should reset this field, and entirely omitting this field would have a distinct meaning.

Reply:
Removed 👍